autopilot/contractor/contractor.go

```diff
@@ -268,12 +237,7 @@ func (c *Contractor) refreshContract(ctx *mCtx, contract contract, host api.Host
 	}
 
 	// calculate the renter funds
-	var renterFunds types.Currency
-	if contract.IsOutOfFunds() {
-		renterFunds = c.refreshFundingEstimate(contract, logger)
-	} else {
-		renterFunds = rev.RenterFunds // don't increase funds
-	}
+	renterFunds := c.refreshFundingEstimate(contract, logger)
```

Comment:
Should we `max` this with `rev.RenterFunds`?

Reply:
The refresh RPC adds funds on top of the renter funds already in the contract. So the available funds after the refresh will never be below `RenterFunds` anyway.